### PR TITLE
*: Add TiDB Playground portal in README.md

### PR DESCRIPTION
**Pull request by ChenlingLu**
Add Playground link to the 'What is TiDB' section

### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by ChenlingLu** [comment]
@winkyao PTAL

**Review by dveeden — COMMENTED** [review]
Left one comment to rephrase the sentence a bit

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ TiDB (/’taɪdiːbi:/, "Ti" stands for Titanium) is an open-source distributed
 - [Architecture](#architecture)
 - [MySQL compatibility](https://docs.pingcap.com/tidb/stable/mysql-compatibility)
 
+See what TiDB is capable ONLINE, go to [>_TiDB Playground](https://play.tidbcloud.com/?utm_source=github&utm_medium=tidb_readme).
```
Comment (hawkingrei) [inline README.md:20]:
```suggestion
See what TiDB is capable ONLINE, go to [TiDB Playground](https://play.tidbcloud.com/?utm_source=github&utm_medium=tidb_readme).
```

**Review by winkyao — APPROVED** [review]
LGTM

**Review by hawkingrei — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ TiDB (/’taɪdiːbi:/, "Ti" stands for Titanium) is an open-source distributed
 - [Architecture](#architecture)
 - [MySQL compatibility](https://docs.pingcap.com/tidb/stable/mysql-compatibility)
 
+See what TiDB is capable ONLINE at [TiDB Playground](https://play.tidbcloud.com/?utm_source=github&utm_medium=tidb_readme).
```
Comment (dveeden) [inline README.md:20]: Shouldn't it be "See what TiDB is capable **of** ONLINE at ..."? (

**Pull request merged**